thirdparty/libsodium/Makefile

```diff
@@ -6,7 +6,7 @@ pvers := $(SODIUM_VERSION)
 pname := libsodium
 pfile := $(pname)-$(pvers).tar.gz
 psrc_file := $(SRC_DIR)/$(pfile)
-purl := https://download.libsodium.org/libsodium/releases/$(pfile)
+purl := https://download.libsodium.org/libsodium/releases/old/unsupported/$(pfile)
```

Comment:
Sigh, this is the wrong "fix"... we need to upgrade to a supported/recent libsodium.

Reply:
Yes, that's what I noted in the commit message :-) But this allows one to at least do a full-tree rebuild.

I opened [bug 108832](https://bugzilla.zimbra.com/show_bug.cgi?id=108832) regarding the upgrade.